CHANGES.rst

```diff
@@ -20,12 +20,16 @@ New Features
 
   - A class hierarchy was added to allow the representation layer to store
     differentials (i.e., finite derivatives) of coordinates.  This is intended
-    to enable support for velocities in coordinate frames down the road. [#5871]
+    to enable support for velocities in coordinate frames. [#5871]
 
   - ``replicate_without_data`` and ``replicate`` methods were added to
     coordinate frames that allow copying an existing frame object with various
     reference or copy behaviors and possibly overriding frame attributes. [#6182]
 
+  - The ``Representation`` class instances can now contain ``Differential``
```

Comment:
I think these shouldn't be in backticks because they're not actual class names... Could either do `BaseRepresentation` in backticks or just leave off the backticks and lower-case them to not be referncing sepcific class names